components/basic_widgets/SmallDropDown.vue

```diff
@@ -105,6 +107,11 @@ export default {
     this.unsubscribe();
   },
   methods: {
+    update_options_list() {
+      this.options_list = this.dropdown_options.filter((option) => {
+        if (option !== this.chosen_option) return option;
+      });
```

Comment:
This can just be `this.dropdown.filter(option => option !== this.chosen_option)`